pandas/tools/plotting.py

```diff
@@ -217,10 +217,18 @@ def _maybe_valid_colors(colors):
         # check whether each character can be convertable to colors
         maybe_color_cycle = _maybe_valid_colors(list(colors))
         if maybe_single_color and maybe_color_cycle and len(colors) > 1:
-            msg = ("'{0}' can be parsed as both single color and "
-                   "color cycle. Specify each color using a list "
-                   "like ['{0}'] or {1}")
-            raise ValueError(msg.format(colors, list(colors)))
+            # Special case for single str 'CN' match and convert to hex
+            # for supporting matplotlib < 2.0.0
+            if re.match(r'C[0-9]', colors) and len(colors) == 2:
+                prop_cycler = plt.rcParams['axes.prop_cycle']
+                hex_color = prop_cycler.by_key().get('color', ['k'])
```

Comment:
I think this prop_cycler is also only supported for matplotlib versions >= 1.5?

See here how it is handled at another location: https://github.com/GuessWhoSamFoo/pandas/blob/e2497222ecf64e07f4bd7873a1b860021b0b605c/pandas/tools/plotting.py#L185-L190